vipps-login-api-howitworks.md

```diff
@@ -20,23 +20,25 @@ A user chooses to log in with Vipps on a merchant’s site.
 
 ### 2. The Vipps landing page
 
-If the login was started on a desktop device the user will be sent to the Vipps landing page. The user enters their phone number, and can choose to be remembered in the browser for a quicker login the next time. The user is then prompted to log in to Vipps, and follow the instructions in the app.
+If the login was started on a desktop device the user will be sent to the Vipps landing page. If the user is not remembered in browser from earlier, the user enters her phone number, and can choose to be remembered in the browser for a quicker login the next time. The "Remember me in browser" option applies to the browser, independent of merchants. This option is default off on desktop devices.
```

Comment:
I agree with having more information about browser recognition here, but I think the wording is a bit unclear.
How about:
```suggestion
If the login was started on a desktop device the user will be sent to the Vipps landing page if the user has not previously opted in to having her browser remembered. The user enters her phone number, and can choose to be remembered in the browser for a quicker login the next time. The "Remember me in browser" option applies to the browser, independent of merchants. This option defaults to off for desktop devices.
```